Examples/Cocoapods/SpeechToText/README.md

```diff
@@ -0,0 +1,52 @@
+# Cloud Speech-to-Text Streaming gRPC Swift Sample
+
+This app demonstrates how to make streaming gRPC connections to the [Google Cloud Speech-to-Text API](https://cloud.google.com/speech-to-text/) to recognize speech in recorded audio.
+
+## Prerequisites
+- An API key for the Cloud Speech-to-Text API 
+- An OSX machine or emulator
+- [Xcode 10][xcode] or later
+- [Cocoapods][cocoapods] version 1.0 or later
+
+## Quickstart
+- Clone this repo and `cd` into this directory.
+- Run `./COMPILE-PROTOS.sh` to generate the Protocol Buffer and gRPC support files. Note that this requires protoc and the protoc-gen-swift and protoc-gen-swiftgrpc plugins. You can get the plugins by running `swift build` in the root of the grpc-swift repository.
```

Comment:
nit: add `protoc` to the prerequisites

Reply:
Do the plugins to be statically linked? i.e. `make plugins` instead of `swift build`?